scripts/ci/github/check_labels.sh

```diff
@@ -18,24 +18,21 @@ ensure_labels() {
 # Must have one of the following labels
 releasenotes_labels=(
   'B0-silent'
-  'B1-releasenotes'
-  'B7-runtimenoteworthy'
+  'B1-note_worthy'
 )
 
 # Must be an ordered list of priorities, lowest first
 priority_labels=(
-  'C1-low 📌'
-  'C3-medium 📣'
-  'C7-high ❗️'
-  'C9-critical ‼️'
+  'C1-low'
+  'C3-medium'
+  'C7-high'
+  'C9-critical'
```

Comment:
```suggestion
  'C7-critical'
```
Looks like this should be only 'C7-critical' as per table https://www.notion.so/paritytechnologies/3e034a92359149199a33cfe85d17f611?v=54434c7bb30b4952a98280d1ff839249

Reply:
Thanks, also fixed `C7-high` -> `C5-high`